src/renderers/client/types/@nteract/transform-dataresource.d.ts

```
@@ -0,0 +1,4 @@
+declare module '@nteract/transform-dataresource' {
+    export = index;
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)